### ISSUE TITLE
RFLINK Gateway USB not responding since update 2020.2

### ISSUE THREAD

**Issue by emoncuit** [domoticz/domoticz#4183]
Hello,
Since I updated DOMOTICZ from V3 / Raspbian Stretch to 2020.2 / Raspbian Buster, the RFLINK connexion with DOMOTICZ is unstable. 
After running about 24 hours, the sensors can no longer be reached by the RFLINK Gateway USB. It connected on /dev/ttyACM0 port. 
The workaround I found is to refreh the hardware :  in the hardware menu, I have to select my gateway, reselect the port and update it. And after it works again for about 24 hours!
Do you know this issue?

**Comment by gizmocuz** [comment]
Running fine here on a USB port

**Comment by emoncuit** [comment]
Hello Gizmocuz, what do you advice me to fix my issue?

**Comment by gizmocuz** [comment]
You are using ttyACM0 ? How did you connect the RFLink ?
Can't you use a normal USB port ? Try this first, make sure your power supply is 2 Amps as well
Do you receive things like a temperature sensor ? If yes, you could also enable a hardware timeout in the hardware setup of 1 hour, but it should not be nececery

**Comment by emoncuit** [comment]
The RFlink is connected to an USB port on my RaspBerry 3. The port mounted is /dev/ttyACM0... 
Yes I receive well like temperature but during a limited time. When it's wrong I "refresh" the hardware.

**Comment by gizmocuz** [comment]
Double check your power supply, try a fresh Raspbian (Buster) installation, and if that does not work, enable the hardware timeout

**Comment by emoncuit** [comment]
Ok @gizmocuz I will try a fresh install.
Thanks

**Comment by emoncuit** [comment]
So, I made a fresh install of Buster and Domoticz 2020.2. I have updated the firmware of the Raspberry 3. I also saw that I had voltage issues. So I have replaced the power cable. It fixed the problem. I don't have any more voltage issues in the syslog. But after 3 days, I had to "re" update the gateway because domoticz wasn't working. The value of temperature sensors were not updated and I couldn't turn on or off my devices ! It's very strange...
